kubernetes/init/setenv.sh

```diff
@@ -34,6 +34,6 @@ if [ -z "$KUBERNETES_NAMESPACE" ]; then
   export KUBERNETES_NAMESPACE='default'
 fi
 if [ -z "$SKIPPER_SERVER_IMAGE" ]; then
-  export SKIPPER_SERVER_IMAGE='springcloud/spring-cloud-skipper-server:1.0.0.RELEASE'
+  export SKIPPER_SERVER_IMAGE='springcloud/spring-cloud-skipper-server:latest'
```

Comment:
I think `latest` is the default and hence it is not required. No harm specifying it explicitly as well :-)